### ISSUE TITLE
Issue with Mod Finder

### ISSUE THREAD

**Issue by Seinra** [SlyDevil/Centipede-Corruptor#1]
ModFinder gives you an error message when trying to install this mod from there, and if I download your mod from GitHub and then I drag the zip to ModFinder to install it, the mod ends with another name:

![image](https://github.com/SlyDevil/Centipede-Corruptor/assets/7744272/a4c74fe6-74fc-4d7b-89d2-8ae8f328de01)
![image](https://github.com/SlyDevil/Centipede-Corruptor/assets/7744272/82aebf2e-e8c5-4adf-bffd-b48f6eaf9bc9)

**Comment by SlyDevil** [comment]
Good catch.

I wrote the incorrect ID when I made the pull request to modfinder to add my mod to its list, so it gets a mod ID mismatch between the ID I put on github and the ID(UniqueName) contained in the OwlcatModificationManifest json.
I've made a new request to fix this error (and the same error on 2 other mods); it should be fixed within a day depending on other people's schedules.

The name difference has also been *rectified* because I wouldn't want github installers to miss out on some extra alliteration in their lives.
